### ISSUE TITLE
Update functions for backing up, printing, and exporting so file extensions aren't duplicated

### ISSUE THREAD

**Issue by RRohrssen** [unfoldingWord-dev/ts-desktop#479]
Make sure these boxes are checked before submitting your issue -- thank you!

- [X] Check that you are using the [latest version](https://github.com/unfoldingWord-dev/ts-desktop/releases/latest) of the app.
- [ ] Check that a similar issue has not already been opened.

**Comment by RRohrssen** [comment]
See https://github.com/unfoldingWord-dev/ts-requirements/issues/54

**Comment by cdwhitfield73** [comment]
@RRohrssen I think you misunderstood the issue that Chris K. was presenting.  It's not that he wants the app to add the number, it's that when he added a number, the app was adding an extra extension on to the file name that already had the extension.  This is the problem that I fixed.  I suppose it would be possible to have the app check for an existing file and if it finds one, add a number to the "suggested" file name of the new file, but again, this is not the issue he was bringing up, and there is a confirmation step in place when you try to overwrite, so I think we are good for now.  Just wanted to clarify the issue and the fix (and I noticed you asked Joel about this in android and the title of the issue is confusing)
